services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -47,11 +47,11 @@ class FeedbackHistory < ActiveRecord::Base
   ]
 
   before_validation :confirm_prompt_type, on: :create
-
+  has_many :feedback_history_ratings
   belongs_to :activity_session, foreign_key: :activity_session_uid, primary_key: :uid
   belongs_to :prompt, polymorphic: true
   belongs_to :concept, foreign_key: :concept_uid, primary_key: :uid
-
+  belongs_to :rule, class_name: Comprehension::Rule,  foreign_key: :rule_uid
```

Comment:
Not a game-breaker, but when we originally spec'ed the `FeedbackHistory` feature, one of the goals was for it to be generic enough to work with non-Comprehension activities.  We've never actually talked about what that would mean, so a `belongs_to` here is probably fine, just wanted to call it out.

Reply:
Actually, after writing this line I opted to use a where query, so I can remove this!